### PR TITLE
Added, and adjusted fursuits and their protection values and rarity

### PR DESCRIPTION
**Pull request by Holli-Git**
#### Summary
Content "Adds more fursuits, changes protection values, and changes rarity"

#### Purpose of change

Adds two new suits, the dragon and feline suit for some more variety. Also changes the protection values of the wolf suit and dino suit, and by extension the dragon and feline suit. Also changes the rarity spawns to more reflect the commonality of the suits, with the feline and wolf being more common than the dragon and the dinosaur.  

#### Describe the solution

Changes the material thickness of the suits so that the protection values aren't super great. 

#### Testing

Spawned in the suits via debug to check their armour and encumbrance values. 

#### Additional context

image
image

### CODE REVIEW

**Comment by actual-nh** [comment]
For the PR summary, unfortunately one can't put in more than one category - Content is probably the right one here. (The PR summaries are used to put together the (first draft of the) changelogs.)

**Comment by Paprika-H** [comment]
> For the PR summary, unfortunately one can't put in more than one category - Content is probably the right one here. (The PR summaries are used to put together the (first draft of the) changelogs.)

Ah, alright! Fixed it, thanks for bringing that to my attention!

File: data/json/items/armor/suits_clothes.json [hunk 1]
```diff
@@ -78,6 +78,59 @@
...
+    "material_thickness": 3,
+    "environmental_protection": 2,
+    "flags": [ "OUTER" ]
+  },
+  {
+    "id": "felinesuit",
+    "repairs_like": "trenchcoat",
+    "type": "ARMOR",
+    "name": { "str": "feline suit" },
+    "description": "A furred suit, made to resemble an anthropomorphic feline of indistinguishable breed. Still, a decent source of fur.",
```
Comment (casswedson) [inline data/json/items/armor/suits_clothes.json:113]:
```suggestion
    "description": "A furred suit, made to resemble an anthropomorphic feline of indistinguishable breed.  Still, a decent source of fur.",
```

File: data/json/items/armor/suits_clothes.json [hunk 2]
```diff
@@ -78,6 +78,59 @@
     "environmental_protection": 2,
     "flags": [ "OUTER" ]
   },
+  {
+    "id": "dragonsuit",
+    "repairs_like": "trenchcoat",
+    "type": "ARMOR",
+    "name": { "str": "dragon suit" },
+    "description": "A cartoonish recreation of a western dragon remade into a suit. Effectively useless nowadays, unless you think the mutants will take you as their own if you wear this. The wings may be a bit more encumbering though.",
```
Comment (casswedson) [inline data/json/items/armor/suits_clothes.json:86]:
```suggestion
    "description": "A cartoonish recreation of a western dragon remade into a suit.  Effectively useless nowadays, unless you think the mutants will take you as their own if you wear this.  The wings may be a bit more encumbering though.",
```

File: data/json/items/armor/suits_clothes.json [hunk 3]
```diff
@@ -78,6 +78,59 @@
     "environmental_protection": 2,
     "flags": [ "OUTER" ]
   },
+  {
+    "id": "dragonsuit",
+    "repairs_like": "trenchcoat",
+    "type": "ARMOR",
+    "name": { "str": "dragon suit" },
+    "description": "A cartoonish recreation of a western dragon remade into a suit.  Effectively useless nowadays, unless you think the mutants will take you as their own if you wear this.  The wings may be a bit more encumbering though.",
```
Comment (actual-nh) [inline data/json/items/armor/suits_clothes.json:86]:
```suggestion
    "description": "A cartoonish recreation of a western dragon remade into a suit.  Effectively useless nowadays, unless you think the mutants will take you as their own if you wear this.  The wings may be a bit too encumbering though.",
```

**Review by actual-nh — COMMENTED** [review]
Looks good!

**Comment by actual-nh** [comment]
The dragon and dino suits would be thicker than the feline and wolf ones?

**Pull request merged**